### DISCUSSION TITLE
I got a problem when i use vllm to run models on Gaudi2 docker images, need help

### DISCUSSION THREAD

**Discussion by DylanLeeee** [HabanaAI/vllm-fork#535] [Q&A]
ValueError: custom_op(...)(func): Parameter w1_scale has a default value; this is not supported. If you want to use default values then create a function with default values that calls the CustomOp Got func with signature (hidden_states: torch.Tensor, w1: torch.Tensor, w2: torch.Tensor, topk_weights: torch.Tensor, topk_ids: torch.Tensor, use_fp8_w8a8: bool, use_int8_w8a16: bool, w1_scale: Optional[torch.Tensor] = None, w2_scale: Optional[torch.Tensor] = None, a1_scale: Optional[torch.Tensor] = None, a2_scale: Optional[torch.Tensor] = None) -> None)

**Comment by michalkuligowski** [comment]
Hi @DylanLeeee, Can you share the command that you are running when this error occurs?
